markdown/ch7.markdown

```diff
@@ -69,8 +69,8 @@ and others examine the entire parameter space.
 ## Optimization in SciPy: `scipy.optimize`
 
 In the rest of this chapter, we are going to use SciPy's `optimize` module to
-align two images. Applications of image alignment or *registration* include
-panorama stitching, combination of multi-modal brain scans, super-resolution
+align two images. Applications of image alignment, or *registration*, include
+panorama stitching, combination of brain scans, super-resolution
```

Comment:
Well, actually that's a thing (taking many different brains and aligning them to get an "average" brain atlas). How do you feel about "combination of different brain scans"? I wanted to keep the language simple here.